test/unit/lib/plugins/aws/package/compile/events/kafka.test.js

```diff
@@ -306,14 +311,14 @@ describe('test/unit/lib/plugins/aws/package/compile/events/kafka.test.js', () =>
       await runCompileEventSourceMappingTest(eventConfig);
     });
 
-    it('should correctly compile EventSourceMapping resource properties for SASL_SCRAM_256_AUTH', async () => {
+    it('should correctly compile EventSourceMapping resource properties for SASL_SCRAM_512_AUTH', async () => {
```

Comment:
fixed this; there was a duplicate test 😎

Reply:
great catch :+1:

Reply:
cheers 🚀